lib/execution_engine2/utils/SDKMethodRunner.py

```diff
@@ -528,13 +558,25 @@ def finish_job(self, job_id, ctx, error_message=None):
             job.errormsg = error_message
             self.get_mongo_util().update_job_status(job_id=job_id, status=Status.error.value)
         else:
+            if not job_output:
+                raise ValueError("Missing job output for finished job")
+
+            output = JobOutput()
+            output.version = job_output.get("version")
```

Comment:
@bio-boris can you verify `output = self._watch(config)` does have 'version', 'id' and 'result' before calling finish_job?

https://github.com/kbase/JobRunner/blob/ee2/JobRunner/JobRunner.py#L292